client/my-sites/customer-home/cards/launchpad/index.tsx

```diff
@@ -43,6 +44,12 @@ const CustomerHomeLaunchpad: FC< CustomerHomeLaunchpadProps > = ( {
 		return null;
 	}
 
+	const hasChecklist = checklist !== undefined && checklist !== null;
```

Comment:
You can use {isLoading} from the `useSortedLaunchpadTasks` hook to check if the interface is ready to be shown. 

or

`hasChecklist = !!checklist`

Reply:
It looks like the behaviour for `isLoading` vs `isFetching` vs `isRefetching` from `useQuery` is relatively complicated. I think I will stick with the checks as-is. We could also switch to `!!`, but I prefer the more explicit checks as I believe both values are possible.